src/algorithms/tracking/CKFTracking.cc

```diff
@@ -392,14 +436,22 @@ namespace eicrecon {
                     } else {
                         auto meas2D = meas2Ds[srclink_index];
                         if (typeFlags.test(Acts::TrackStateFlag::MeasurementFlag)) {
+                          #if EDM4EIC_VERSION_MAJOR >= 5
+                            //track.addToMeasurementHits(meas2D);
```

Comment:
Don't we still need this in edm4eic v5? Or am I missing something:
https://github.com/eic/EDM4eic/blob/773bf2628711dfed082b12416f9179a0ddf3a67e/edm4eic.yaml#L416

Reply:
We don't yet fill a edm4eic::Track in CKFTracking.

Reply:
Ah, you're right I missed that. So I guess that is being left to a future PR then

Reply:
Yeah. I think that's best.